### ISSUE TITLE
In v5.3.7 /taxii2/root/collections/:id/objects no longer works.

### ISSUE THREAD

**Issue by zanoni23** [LuatixHQ/opencti#2291]
## Description

We’ve just upgraded from 5.1.4 -> 5.2.4 -> 5.3.7 and found an issue with taxii2.
We can successfully do
```
GET /taxii2/root/collections
    /taxii2/root/collections/:id
    /taxii2/root/collections/:id/manifest 
```
and using an object-id from the manifest can successfully do 
`GET /taxii2/root/collections/:id/objects/:object_id`

But if we try  
`GET /taxii2/root/collections/:id/objects `
we get http 500 after a long pause (about 11 seconds) and this error message:

`{"title":"Unsupported operation","error_code":"UnsupportedError","description":"Cannot convert relation without a resolved to: 471019a5-01c3-46c8-b187-96c9d940cf51","http_status":500,"details":{"category":"business"}}`

but nothing logged in the opencti-platform log (at debug log level).
I would test this via demo.opencti but it seems I don’t have permission to access /taxii2/root

## Environment

1. OS - docker containers in k8s
2. OpenCTI version: 5.3.7
3. OpenCTI client: Postman, Firefox, etc
4. Other environment details:   AWS Opensearch 1.0  or 1.2

## Reproducible Steps

Go to https://site/taxii2/root/collections/ and get a collection id, and then use the collection id
Go to https://site/taxii2/root/collections/id/objects

## Expected Output

Something like:
`{"more":false,"next":"WzE2NTQyMDIyNTgwNjYsIjM0ZjdkYTEyLWJmNzItNDVkYi05NjQ3LTY3MGZlNzU0NzZjMSJd","objects":[{"id":"indicator--f6d25727-8468-5214-be8d-410fb2e8941a","pattern_type":"stix","pattern":"[ipv4-addr:value = '88.88.88.88']","name":"Athena - Test Feed","valid_from":"2022-05-02T20:37:00.000Z","valid_until":"2022-06-02T20:37:00.000Z","confidence":75,"x_opencti_detection":false,"x_opencti_main_observable_type":"IPv4-Addr","x_opencti_score":50,"spec_version":"2.1","created_at":"2022-05-02T20:39:57.548Z","updated_at":"2022-06-02T20:37:38.066Z","revoked":true,"lang":"en","created":"2022-05-02T20:39:57.548Z","modified":"2022-06-02T20:37:38.066Z","created_by_ref":"identity--d3218e19-0809-5b82-bbbf-e9caf9ab22d9","object_marking_refs":["marking-definition--f88d31f6-486f-44da-b317-01333bde0b82"],"kill_chain_phases":[{"kill_chain_name":"mitre-pre-attack","phase_name":"technical-information-gathering"}],"x_opencti_id":"34f7da12-bf72-45db-9647-670fe75476c1","type":"indicator","x_opencti_type":"Indicator"}]}`

## Actual Output

`{"title":"Unsupported operation","error_code":"UnsupportedError","description":"Cannot convert relation without a resolved to: 471019a5-01c3-46c8-b187-96c9d940cf51","http_status":500,"details":{"category":"business"}}`

## Additional information

Cannot find anything in logs
In Opensearch I enabled slow search logs on half a dozen indices (threshold >2 seconds), and it hasn't logged anything, so the long pause maybe inside the opencti-platform container.

**Comment by richard-julien** [comment]
Hi @zanoni23 ,

After investigation, this situation can happen if your user (THL:WHITE) have access to a relation (TLP:WHITE) that linked an entity with a marking that you cannot access (TLP:RED). 
I will work on that.
Can you check on your side if you are on this situation?

**Comment by zanoni23** [comment]
We see that issue even when we are logged in as Admin and so should be able to see everything within the platform, regardless of what TLP the entry should be. The test feeds in dev contain TLP:WHITE indicators, so should be visible to anyone.
Another user messaged me on the Luatix Slack asking about this, as he has the issue too (though his wasn't after an upgrade, but just setting up the TAXII feed).

**Comment by richard-julien** [comment]
Just to be clear about how the platform behave today.

- Bypassing markings is only possible with "Bypass all capabilities" capability granted for the user.
- TLP:WHITE is no different than other markings, if the user is not part of a group that have access to TLP:WHITE, the information will not be visible. 
- This error is due to a inconsistency when generating the stix information from the database, it could happen in different part of OpenCTI. Inconsistency can be due to a access right inconsistency or a database inconsistency (more problematic obsiously)

Of course this kind of inconsistency should not brake the feature, currently working on it :)

**Comment by zanoni23** [comment]
We turned the bypass on for the admin account and it worked with a standard feed (no HTTP 500 error). But when we added an AlienVault OTX feed it didn't work - 500 error.


Is this related? Our incoming feed using the TAXII2 connector has also stopped working.  An AlienVault feed is still working fine. The taxii2 connector container logs (at DEBUG level):
```
INFO:root:Sending Bundle to server with '100' objects
INFO:root:Run Complete. Sleeping until next run in 1 hours
```
RabbitMQ doesn't log anything unusual. There are no indicators or data in the platform from the feed. The gui says the connector is active with the last update being the current time. A listen and push queue for the connector are in RabbitMQ.

Whereas the feed on the production opencti setup which is still on version 5.1.4  has pika.adapters.utils log entries (which refer to RabbitMQ connections) logged after "Sending Bundle" and has "Bundle has been sent" messages before the "Run Complete" message.
